docs/source/runtime_troubles.rst

```diff
@@ -0,0 +1,311 @@
+.. _runtime_troubles:
+
+Annotation issues at runtime
+============================
+
+Idiomatic use of type annotations can sometimes run up against what a given
+version of Python considers legal code. This section describes these scenarios
+and explains how to get your code running again. Generally speaking, we have
+three tools at our disposal:
+
+* For Python 3.7 through 3.9, use of ``from __future__ import annotations``
```

Comment:
Should string literals be in this list?

Reply:
Yeah, I put string literals in the list because they're the easiest way to deal with issues in 3.6.